pkg/api/mlflow/service/metric/service.go

```diff
@@ -19,102 +19,62 @@ import (
 )
 
 func GetMetricHistory(c *fiber.Ctx) error {
-	id := c.Query("run_id", c.Query("run_uuid"))
-	key := c.Query("metric_key")
+	req := request.GetMetricHistoryRequest{}
+	if err := c.QueryParser(&req); err != nil {
+		return api.NewBadRequestError(err.Error())
+	}
 
-	log.Debugf("GetMetricHistory request: run_id=%q, metric_key=%q", id, key)
+	log.Debugf("GetMetricHistory request: run_id=%q, run_uuid=%q metric_key=%q", req.RunID, req.RunUUID, req.MetricKey)
```

Comment:
We could now just print `req` like we do in other places

Reply:
probably yes.